### ISSUE TITLE
Missing Relationship causes profile page to raise error

### ISSUE THREAD

**Issue by aepyornis** [public-accountability/littlesis-rails#195]
```
INFO -- : Started GET "/entities/52680" 
INFO -- : Processing by EntitiesController#show as HTML
INFO -- :   Parameters: {"id"=>"52680"}
INFO -- :   Rendered shared/_facebook_sdk.html.erb (0.0ms)
INFO -- :   Rendered entities/_header.html.erb (0.1ms)
INFO -- :   Rendered shared/_entity_history.html.erb (2.8ms)
INFO -- :   Rendered entities/_actions.html.erb (3.4ms)
INFO -- :   Rendered entities/_summary.html.erb (0.4ms)
INFO -- :   Rendered entities/_tabs.html.erb (0.2ms)
 INFO -- : Read fragment views/relationship_page/entity/52680 1491149878/bb15ba1b131fe4676f01f172400345e4 (0.2ms)
INFO -- :   Rendered entities/show.html.erb within layouts/application (72.4ms)
INFO -- : Completed 500 Internal Server Error in 87ms (ActiveRecord: 21.3ms)
 FATAL -- : ActionView::Template::Error (undefined method `amount' for nil:NilClass):
    19:             <% if params[:relationships] == nil %>
    20:
    21:                 <% cache ['relationship_page', @entity.alt_cache_key] do  %>
    22:                     <% @links = SortedLinks.new(@entity) %>
    23:                     <% section_order(@entity).each do |section| %>
    24:                         <% links_group = @links.send(section) %>
    25:                         <%= section_heading(links_group) %>
  app/utility/links_group.rb:28:in `block in order'
  app/utility/links_group.rb:28:in `sort'
  app/utility/links_group.rb:28:in `order'
  app/utility/links_group.rb:10:in `initialize'
  app/utility/sorted_links.rb:159:in `new'
  app/utility/sorted_links.rb:159:in `create_donation_subgroups'
  app/utility/sorted_links.rb:107:in `create_subgroups'
  app/utility/sorted_links.rb:65:in `initialize'
  app/views/entities/show.html.erb:22:in `new'
  app/views/entities/show.html.erb:22:in `block in _app_views_entities_show_html_erb__2590858514928522330_52430160'
  app/views/entities/show.html.erb:21:in `_app_views_entities_show_html_erb__2590858514928522330_52430160'

```

**Comment by aepyornis** [comment]
Links exist and should have been deleted.

this will delete them:

```
Entity.find( <ID> ).links.to_a.select { |l| l.relationship.nil? }.each { |l| l.destroy }
```

**Comment by aepyornis** [comment]
fixed by #207
